internal/cmd/gocdk/serve.go

```diff
@@ -16,10 +16,213 @@ package main
 
 import (
 	"context"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"net/url"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"time"
 
 	"golang.org/x/xerrors"
 )
 
 func serve(ctx context.Context, pctx *processContext, args []string) error {
-	return xerrors.New("not implemented")
+	f := newFlagSet(pctx, "serve")
+	if err := f.Parse(args); xerrors.Is(err, flag.ErrHelp) {
+		return nil
+	} else if err != nil {
+		return usagef("gocdk serve: %w", err)
+	}
+	if f.NArg() > 1 {
+		return usagef("gocdk serve [BIOME]")
+	}
+	biome := f.Arg(0)
+	if biome == "" {
+		biome = "dev"
+	}
+
```

Comment:
after the intended biome name is determined, we need to check that (a) it exists and (b) that it has a config.json file in which `serve_enabled = true`

Reply:
Ack. I spent a bit of time after this comment trying to implement, and the resulting code would a fair bit of heft to an already hefty PR. I've opened #1833 for this.